packages/builder/src/builtin-tasks/run.ts

```diff
@@ -17,17 +22,29 @@ function filterNonExistent (scripts: string[]): string[] {
 }
 
 export async function runMultipleScripts (runtimeEnv: AlgobRuntimeEnv,
-  scriptNames: string[],
-  log: (...args: unknown[]) => unknown): Promise<void> {
+                                          scriptNames: string[],
+                                          log: (...args: unknown[]) => unknown,
```

Comment:
Why passing `log` here? If it's for tests, than probably it's better to use `debug` module.

Also, the function should return `void`. I prefer to use `any` instead of `unknown` in this context, because we know that it can be any argument.

Reply:
This is only because linter complains.

Reply:
for `any` - there is a reason for that. You can mute it with appropriate comment (eslint-disable ....)

Reply:
So why we are using `log` as a function argument?

Reply:
This is because `deploy` and `run` tasks need to log during verbose mode. It's a very minor thing, log has a tag so it preserves that tag.

Reply:
Tasks create the method themselves. Log doesn't need to be passed as a parameter anymore.